data/darktableconfig.xml.in

```diff
@@ -2344,6 +2344,33 @@
     <shortdescription>Geotagging search URL</shortdescription>
     <longdescription>this can be changed when the default OpenStreetMap search URL is broken</longdescription>
   </dtconfig>
+  <dtconfig prefs="darkroom" section="interface">
+    <name>plugins/darkroom/temperature/colored_sliders</name>
+    <type>
+      <enum>
+        <option>no color</option>
+        <option>illuminant color</option>
+        <option>effect emulation</option>
+      </enum>
+    </type>
+    <default>no color</default>
+    <shortdescription>white balance slider colors</shortdescription>
+    <longdescription>visual indication of temperature adjustments. in 'illuminant color' mode slider colors correspond to the color of the light source, in 'effect emulation' slider colors correspond to the effect the adjustment would have on the scene</longdescription>
```

Comment:
Following from discussions on IRC... This might be better with some line breaks in it and for the word "correspond" to change to the word "represent" to reflect that the sliders aren't "the same as" the light source / scene effect, but rather just a shortcut way to reference those things.